[2019-04-29 08:06:22,492] {models.py:1359} INFO - Dependencies all met for <TaskInstance: s3_buckkeyright_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [queued]>
[2019-04-29 08:06:22,493] {models.py:1359} INFO - Dependencies all met for <TaskInstance: s3_buckkeyright_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [queued]>
[2019-04-29 08:06:22,493] {models.py:1571} INFO - 
--------------------------------------------------------------------------------
Starting attempt 2 of 2
--------------------------------------------------------------------------------

[2019-04-29 08:06:22,502] {models.py:1593} INFO - Executing <Task(S3KeySensor): s3_file_test> on 2019-04-28T00:00:00+00:00
[2019-04-29 08:06:22,502] {base_task_runner.py:118} INFO - Running: ['bash', '-c', 'airflow run s3_buckkeyright_sensor s3_file_test 2019-04-28T00:00:00+00:00 --job_id 104 --raw -sd DAGS_FOLDER/AWS_Challenge.py --cfg_path /tmp/tmpn443140b']
[2019-04-29 08:06:22,950] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test [2019-04-29 08:06:22,950] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-29 08:06:23,098] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test [2019-04-29 08:06:23,097] {models.py:273} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/AWS_Challenge.py
[2019-04-29 08:06:23,131] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test [2019-04-29 08:06:23,131] {cli.py:520} INFO - Running <TaskInstance: s3_buckkeyright_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [running]> on host localhost.localdomain
[2019-04-29 08:06:23,173] {s3_key_sensor.py:88} INFO - Poking for key : s3://s3://cscie29-data/pset5/yelp_data//yelp_subset_*.csv
[2019-04-29 08:06:23,190] {logging_mixin.py:95} INFO - [2019-04-29 08:06:23,189] {credentials.py:1046} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2019-04-29 08:06:23,215] {models.py:1788} ERROR - Parameter validation failed:
Invalid bucket name "s3://cscie29-data/pset5/yelp_data/": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$"
Traceback (most recent call last):
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/models.py", line 1657, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/sensors/base_sensor_operator.py", line 97, in execute
    while not self.poke(context):
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/sensors/s3_key_sensor.py", line 91, in poke
    self.bucket_name)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 293, in check_for_wildcard_key
    delimiter=delimiter) is not None
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 310, in get_wildcard_key
    klist = self.list_keys(bucket_name, prefix=prefix, delimiter=delimiter)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 176, in list_keys
    for page in response:
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/paginate.py", line 255, in __iter__
    response = self._make_request(current_kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/paginate.py", line 332, in _make_request
    return self._method(**current_kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 634, in _make_api_call
    api_params, operation_model, context=request_context)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 680, in _convert_to_request_dict
    api_params, operation_model, context)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 712, in _emit_api_params
    params=api_params, model=operation_model, context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 356, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 228, in emit
    return self._emit(event_name, kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 211, in _emit
    response = handler(**kwargs)
  File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/handlers.py", line 223, in validate_bucket_name
    raise ParamValidationError(report=error_msg)
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid bucket name "s3://cscie29-data/pset5/yelp_data/": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$"
[2019-04-29 08:06:23,217] {models.py:1817} INFO - All retries failed; marking task as FAILED
[2019-04-29 08:06:23,230] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test /home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/utils/helpers.py:356: DeprecationWarning: Importing 'BashOperator' directly from 'airflow.operators' has been deprecated. Please import from 'airflow.operators.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
[2019-04-29 08:06:23,230] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   DeprecationWarning)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test Traceback (most recent call last):
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/bin/airflow", line 32, in <module>
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     args.func(args)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return f(*args, **kwargs)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/bin/cli.py", line 526, in run
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     _run(args, dag, ti)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/bin/cli.py", line 445, in _run
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     pool=args.pool,
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/utils/db.py", line 73, in wrapper
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return func(*args, **kwargs)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/models.py", line 1657, in _run_raw_task
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     result = task_copy.execute(context=context)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/sensors/base_sensor_operator.py", line 97, in execute
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     while not self.poke(context):
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/sensors/s3_key_sensor.py", line 91, in poke
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     self.bucket_name)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 293, in check_for_wildcard_key
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     delimiter=delimiter) is not None
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 310, in get_wildcard_key
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     klist = self.list_keys(bucket_name, prefix=prefix, delimiter=delimiter)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/hooks/S3_hook.py", line 176, in list_keys
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     for page in response:
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/paginate.py", line 255, in __iter__
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     response = self._make_request(current_kwargs)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/paginate.py", line 332, in _make_request
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return self._method(**current_kwargs)
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 357, in _api_call
[2019-04-29 08:06:23,233] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return self._make_api_call(operation_name, kwargs)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 634, in _make_api_call
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     api_params, operation_model, context=request_context)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 680, in _convert_to_request_dict
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     api_params, operation_model, context)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/client.py", line 712, in _emit_api_params
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     params=api_params, model=operation_model, context=context)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 356, in emit
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return self._emitter.emit(aliased_event_name, **kwargs)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 228, in emit
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     return self._emit(event_name, kwargs)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/hooks.py", line 211, in _emit
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     response = handler(**kwargs)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test   File "/home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/botocore/handlers.py", line 223, in validate_bucket_name
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test     raise ParamValidationError(report=error_msg)
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test botocore.exceptions.ParamValidationError: Parameter validation failed:
[2019-04-29 08:06:23,234] {base_task_runner.py:101} INFO - Job 104: Subtask s3_file_test Invalid bucket name "s3://cscie29-data/pset5/yelp_data/": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$"
[2019-04-29 08:06:27,502] {logging_mixin.py:95} INFO - [2019-04-29 08:06:27,502] {jobs.py:2527} INFO - Task exited with return code 1
